src/develop/masks.h

```diff
@@ -328,6 +330,8 @@ void dt_masks_form_remove(struct dt_iop_module_t *module, dt_masks_form_t *grp,
 void dt_masks_form_change_opacity(dt_masks_form_t *form, int parentid, int up);
 void dt_masks_form_move(dt_masks_form_t *grp, int formid, int up);
 int dt_masks_form_duplicate(dt_develop_t *dev, int formid);
+/* duplicate the list of forms, replace item in the list with form is the same formid */
```

Comment:
can't parse this one. "with form is the same formid" should it be "with form with the same formid"?